Python/cryptography/Columnar_Cipher.py

```diff
@@ -0,0 +1,101 @@
+'''
+In Columnar Cipher, we are given a plaintext and a key. The plaintext is written out in rows
```

Comment:
Line too long pass your code on linter [here](http://pep8online.com) and fix the lining /styling issues